docs/src/gfp.md

```diff
@@ -60,7 +60,7 @@ Below we describe the functionality that is provided in addition to these.
 
 ```jldoctest
 julia> F = GF(3)
-Finite field of characteristic 3
+Finite field of degree 1 over GF(3)
```

Comment:
I think this could be confusing for some users. Can we adapt the printing for degree 1 extensions to be more similar to the old behavior?

Reply:
The phrase "[Finite|Galois] field of characteristic p" is what we had before. Is ambiguous and confusing. We have to revisit some of the printing later anyway.